src/coreclr/scripts/genRuntimeEventSources.py

```diff
@@ -305,7 +321,29 @@ def generateEnumTypeMap(providerNode):
 
     return typeMap
 
-def generateKeywordsClass(providerNode, outputFile):
+def generateKeywordsClass(providerNode, outputFile, inclusion_list):
+
+    providerName = providerNode.getAttribute("name")
+
+    # Get the events element.
+    for node in providerNode.getElementsByTagName("events"):
+        eventsNode = node
+        break
+
+    # Get the list of event nodes.
+    eventNodes = eventsNode.getElementsByTagName("event")
+
+    # Build the list of used keywords
```

Comment:
This will omit `OverrideAndSuppressNGenEvents`, `StartEnumerationKeyword` and  `EndEnumerationKeyword` keywords in CoreCLR version as well since they are unused. I assume that it is fine.